tests/integrationv2/configuration.py

```diff
@@ -81,7 +81,6 @@
 
     Ciphers.ECDHE_RSA_AES128_SHA,
     Ciphers.ECDHE_RSA_AES256_SHA,
-    Ciphers.ECDHE_RSA_DES_CBC3_SHA,
```

Comment:
Why is this being removed?

Reply:
This is a "weak cipher", and not available in openssl w/o a specific cmd line during compilation. We don't actually test for this in the original integ framework, but I added it here because I saw it listed in the gnuTLS test (which we haven't implemented yet). We may be able to add it back in with that provider.